src/applications/user-testing/disability-benefits-all-claims-conditions-chapter/content/conditions.jsx

```diff
@@ -29,28 +29,45 @@ export const NewConditionDescription = () => (
   </>
 );
 
-const createSecondaryDescriptionString = causedByCondition => {
-  // Just for SecondaryEnhanced demo
-  if (typeof causedByCondition === 'object') {
-    return createSecondaryEnhancedDescriptionString(causedByCondition);
-  }
+const createCauseFollowUpDescriptions = (item, fullData = {}) => {
+  if (!item?.cause) return '';
 
-  return `caused by ${causedByCondition ||
-    'a missing service-connected condition'}`;
-};
+  const { conditions = [], ratedDisabilities = [] } = fullData;
+
+  switch (item.cause) {
+    case 'NEW':
+      return 'caused by an injury, event, disease or exposure during my service';
+
+    case 'SECONDARY': {
+      if (typeof item.causedByCondition === 'object') {
```

Comment:
Is there a way we can avoid doing a `typeof` check here?

Reply:
Thanks for pointing that out. For the prototype, I added the typeof guard mainly as a quick safety check so the UI doesn't blow up should the causedByCondition comes back  as anything unexpected during the demos. When we fold this code into the main app, we'll tighten the types upstream so the extra runtime check can be removed.